shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/java/org/apache/shardingsphere/migration/distsql/handler/query/ShowMigrationListQueryResultSet.java

```diff
@@ -43,13 +44,14 @@ public final class ShowMigrationListQueryResultSet implements DatabaseDistSQLRes
     public void init(final ShardingSphereDatabase database, final SQLStatement sqlStatement) {
         data = JOB_API.list().stream()
                 .map(each -> {
+                    MigrationJobInfo jobInfo = (MigrationJobInfo) each;
                     Collection<Object> result = new LinkedList<>();
```

Comment:
Could we imporve the return type of public API `list()` definition?
So `MigrationJobPublicAPI.list()` could return List of `MigrationJobInfo`, to avoid job info class cast.